microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+	* [CoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [CoreWebView2PermissionSettingCollectionView interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [CoreWebView2PermissionSetting interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [CoreWebView2ControllerOptions2 interface](#)
+	* [CoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [CoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [CoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Profile4 interface](#)
+	* [CoreWebView2Profile4::SetPermissionState method](#)
+	* [CoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [CoreWebView2PermissionSettingCollectionView interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [CoreWebView2PermissionSetting interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [CoreWebView2ControllerOptions2 interface](#)
+	* [CoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [CoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [CoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+
+##### [Win32/C++](#tab/win32cpp)
+* [ICoreWebView2Profile4 interface](#)
+	* [ICoreWebView2Profile4::SetPermissionState method](#)
+	* [ICoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [ICoreWebView2PermissionSettingCollectionView interface](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [ICoreWebView2PermissionSetting interface](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [ICoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [ICoreWebView2ControllerOptions2 interface](#)
+	* [ICoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [ICoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [ICoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [ICoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+---

```

Comment:
```suggestion

---

```
copy of note:
haven't fully tested having tab-set delimiter touching a non-blank line.
Puting blank line above like pattern in this file